src/python/director/startup.py

```diff
@@ -1077,6 +1069,14 @@ def stop(self):
     if (drcargs.args().exo):
         ikPlanner.pushToMatlab = False
 
+# Rotate Multisense image/CAMERA_LEFT if the camera frame is rotated (e.g. for Valkyrie)
+if robotStateModel.getHeadLink():
+    tf = robotStateModel.getLinkFrame(robotStateModel.getHeadLink())
+    roll = transformUtils.rollPitchYawFromTransform(tf)[0]
+    if np.isclose(np.abs(roll), np.pi, atol=1e-1):
+        cameraview.imageManager.setImageRotation180('CAMERA_LEFT')
+
```

Comment:
I think this could by tidier.  This code can be moved up to where cameraview.init() is called, and look like:

cameraview.initImageRotations(robotStateModel)